graphql/schema/schema.graphql

```diff
@@ -177,6 +177,12 @@ type Mutation {
   """Resets the o-counter for a scene to 0. Returns the new value"""
   sceneResetO(id: ID!): Int!
 
+  """Sets the resume time point (if provided) and adds the provided duration to the scene's play duration"""
+  sceneSaveActivity(id: ID!, resume_time: Float, playDuration: Float): Boolean!
+
+  """Increments the watch count for the scene. Returns the new watch count value."""
+  sceneIncrementWatchCount(id: ID!): Int!
```

Comment:
We have an inconsistency in terms (partly my fault). We have `play_count`, `play_duration`, `last_played_at` in the `Scene` type, but `watch` is used here. I'm more partial to `watch` in this context, but I'm not married to the term. One or the other should be changed, though it should be what is the most correct, rather than what is easiest to change.

Reply:
I landed on `play` after digging around to see what variables other sites and apps used. I can update everything to be consistent with `play`.